frontera/contrib/scrapy/converters.py

```diff
@@ -7,6 +7,8 @@
 from frontera.utils.converters import BaseRequestConverter, BaseResponseConverter
 from w3lib.util import to_bytes, to_native_str
 
+from frontera.utils.misc import dict_to_unicode, dict_to_bytes
+
 
 class RequestConverter(BaseRequestConverter):
```

Comment:
I personally think it would be better to solve this issue here for the following reasons.
1. The issue is specific to Scrapy, so it makes sense to solve it in the scrapy converter.
2. Making the conversion in encoder/decoder leads to an unnecessary overhead for other components which use them.